src/test/java/org/opensearch/security/sanity/tests/SecurityRestTestCase.java

```diff
@@ -77,14 +77,20 @@ protected RestClient buildClient(Settings settings, HttpHost[] hosts) throws IOE
                 // create adminDN (super-admin) client
                 File file = new File(getClass().getClassLoader().getResource(CERT_FILE_DIRECTORY).getFile());
                 Path configPath = PathUtils.get(file.toURI()).getParent().toAbsolutePath();
-                return new SecureRestClientBuilder(settings, configPath).setSocketTimeout(60000).build();
+                return new SecureRestClientBuilder(settings, configPath)
+                        .setSocketTimeout(60000)
+                        .setConnectionRequestTimeout(180000)
```

Comment:
Should be fixed by https://github.com/opensearch-project/common-utils/pull/287 but needed manual setting now

Reply:
Should we wait until that issue has been published? or we can create a tracker issue that reverts this change once common-utils publishes an artifact with that change

Reply:
No action is needed: this change is good, it may just be redundant after https://github.com/opensearch-project/common-utils/pull/287 but keeping it is fine, thanks.